vaxrank/cli.py

```diff
@@ -133,17 +135,23 @@ def main(args_list=None):
 
     df.to_csv(args.output_csv, index=False)
 
-    make_ascii_report_from_ranked_vaccine_peptides(
-        ranked_variants_with_vaccine_peptides=ranked_list,
-        mhc_alleles=mhc_alleles,
-        variants=variants,
-        bam_path=args.bam,
-        ascii_report_path=args.output_ascii_report)
-
-    make_html_report_from_ranked_vaccine_peptides(
+    template_data = compute_template_data(
         ranked_variants_with_vaccine_peptides=ranked_list,
         mhc_alleles=mhc_alleles,
         variants=variants,
-        bam_path=args.bam,
-        html_report_path=args.output_html_report,
-        pdf_report_path=args.output_pdf_report)
+        bam_path=args.bam)
+
+    if args.output_ascii_report:
```

Comment:
If no report types are specified, we should fail fast and require one to be (otherwise the user will just wait a bit and Vaxrank will run, but nothing will happen!)